resolwe_bio/processes/expression/mappability.yml

```diff
@@ -39,23 +39,23 @@
     program: |
       echo "MAPPABILITY..."
       echo "UNCOMPRESSING genome..."
-      GENOME_NAME=`basename '{{ genome.fasta.file }}' .fasta.gz`
-      gzip -cd {{ genome.fasta.file }} > "${GENOME_NAME}.fasta"
+      GENOME_NAME=`basename '{{ genome.fasta.file }}' .fasta`
       echo "{\"proc.progress\":0.1,\"proc.rc\":$?}"
 
       echo "BUILDING genome index..."
-      bowtie-build "${GENOME_NAME}.fasta" "${GENOME_NAME}_index"
+      bowtie-build "{{ genome.fasta.file }}" "${GENOME_NAME}_index"
       echo "{\"proc.progress\":0.2,\"proc.rc\":$?}"
 
+      cp "{{ genome.fasta.file }}" .
```

Comment:
This is to copy genome fasta to current working dir. Required by downstream tools.